scripts/default/opensearch-dashboards/build.sh

```diff
@@ -22,6 +26,13 @@ function usage() {
     echo -e "-h help"
 }
 
+
+function cleanup_all() {
+    File_Delete $PLUGIN_PATH
+}
```

Comment:
Thanks @peterzhuamazon where is this `File_Delete` coming from?

Reply:
See the above imported lib of file_management.

Reply:
Got it, its part of https://github.com/opensearch-project/opensearch-build/blob/main/lib/shell/file_management.sh.